indico/web/forms/widgets.py

```diff
@@ -198,6 +198,42 @@ def __call__(self, field, **kwargs):
                                                      search_url=self.search_url, choices=getattr(field, 'choices', []))
 
 
+class LocationWidget(JinjaWidget):
+    """Renders a collection of fields to represent location"""
+
+    def __init__(self):
+        super(LocationWidget, self).__init__('forms/location_widget.html', single_line=True)
+
+    def __call__(self, field, **kwargs):
+        if Config.getInstance().getIsRoomBookingActive():
+            rooms = {loc.name: {'data': self.get_sorted_rooms(loc)} for loc in field.locations}
+            venues = {'data': [{'id': loc.id, 'name': loc.name} for loc in field.locations]}
+        else:
+            rooms = venues = {'data': []}
```

Comment:
I like that when I don't use this, I get comments because I don't use it and now that I do, I still get comments! :disappointed: